packages/datepicker/datepicker.scss

```diff
@@ -155,8 +155,8 @@ $bottom-margin: $component-spacing--large;
         }
 
         caption {
-            font-size: $font-size-3;
-            line-height: $line-height-3;
+            font-size: $font-size-6;
```

Comment:
Greit for nå, men tenker vi bør tokenize font-size. `$font-size-6` bærer ingen informasjon for en leser. Hadde det vært `$font-size--normal` eller `$font-size--large` så sier det litt mer. Samme med linjehøyde.

Reply:
Hva skal da være "normal"? Det må være ganske intuitivt hva som er normal, large, xl etc. dersom det skal ha noen verdi utover å bruke `font-size-6` slik det er nå.